code/modules/mob/living/carbon/human/species_types/ipc.dm

```diff
@@ -0,0 +1,96 @@
+/datum/species/ipc
+	// shitty toasters why do we need them
+	name = "Machine"
+	id = "ipc"
+	say_mod = "states"
+	heatmod = 3 // Went cheap with Aircooling
+	coldmod = 1.5 // Don't put your computer in the freezer.
+	burnmod = 2 // Wiring doesn't hold up to fire well.
+	brutemod = 1.6 // Thin metal, cheap materials.
+	toxmod = 0
+	siemens_coeff = 1.5 // Overload!
+	species_traits = list(NOBREATH,NOBLOOD,RADIMMUNE,VIRUSIMMUNE,NOZOMBIE,EASYDISMEMBER,EASYLIMBATTACHMENT,NOPAIN,NO_BONES,NOTRANSSTING,MUTCOLORS,REVIVESBYHEALING,NOSCAN)
+	mutant_bodyparts = list("ipc_screen", "ipc_antenna", "ipc_chassis")
+	default_features = list("mcolor" = "#7D7D7D", "ipc_screen" = "Static", "ipc_antenna" = "None", "ipc_chassis" = "Morpheus Cyberkinetics(Greyscale)")
+	meat = /obj/item/stack/sheet/plasteel{amount = 5}
+	skinned_type = /obj/item/stack/sheet/metal{amount = 10}
+	male_scream_sound = 'sound/effects/mob_effects/goonstation/robot_scream.ogg'
+	female_scream_sound = 'sound/effects/mob_effects/goonstation/robot_scream.ogg'
+	male_cough_sound = 'sound/effects/mob_effects/machine_cough.ogg'
+	female_cough_sound = 'sound/effects/mob_effects/machine_cough.ogg'
+	male_sneeze_sound = 'sound/effects/mob_effects/machine_sneeze.ogg'
+	female_sneeze_sound = 'sound/effects/mob_effects/machine_sneeze.ogg'
```

Comment:
Odd optimisation question for @FlattestGuitar or @AndrewMontagne ... it'd be easier for them to do the following, no?

`male_scream_sound = 'sound/effects/mob_effects/goonstation/robot_scream.ogg'`
`female_scream_sound = male_scream_sound`

Alternatively on the scream I guess you could check if female is null, and if so play male - so you don't have to worry about setting.... but I guess that relies on female inheriting as null, which it might not.... 🤔

Reply:
You can't do that, because this isn't code that will run, but a class definition. What Ziiro did is optimal. Maybe you could try to fuck with a #define here, but that's overcomplicating it.